File: Escalfament PreNadalenc 2024/ProblemaB.cc
#include <iostream>
using namespace std;

int main() {
    
    int n;
    cin >> n;
    
    //RECORRO FILES (CADA FILA ÉS CAS DE PROVA D'UN MUNDIAL)
    for (int i = 0; i < n; ++i) {
        
        //MIRO EQUIPS QUE TÉ EL MUNDIAL
        int nEquips;
        cin >> nEquips;
        
        //RECORRO ELS EQUIPS PER CADA CONFEDERACIÓ I ELS ACUMULO
        int sumEquipsConfederacio = 0;
        int entrada;
        for (int j = 0; j < 6; ++j) {
            cin >> entrada;
            sumEquipsConfederacio += entrada;
        }
        
        //IMPRIMEIXO EL NOMBRE D'EQUIPS QUE VAN A LA REPESCA INTERCONTINENTAL SI CAL
        cout << nEquips - sumEquipsConfederacio << endl;
    }       

    return 0;
}

File: Escalfament PreNadalenc 2024/ProblemaB.java


import java.util.Scanner;

/**
 * @author santi
 * @fecha  24 ene. 2024
 */
public class ProblemaB {
    public static void main(String[]args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        sc.nextLine(); //consumeixo bufer
        
        //RECORRO FILES (CADA FILA ÉS CAS DE PROVA D'UN MUNDIAL)
        for (int i = 0; i < n; ++i) {
            
            //MIRO EQUIPS QUE TÉ EL MUNDIAL
            int nEquips = sc.nextInt();
            
            //RECORRO ELS EQUIPS PER CADA CONFEDERACIÓ I ELS ACUMULO
            int sumEquipsConfederacio = 0;
            for (int j = 0; j < 6; ++j) {
                sumEquipsConfederacio += sc.nextInt();
            }
            sc.nextLine(); //consumeixo bufer
            
            //IMPRIMEIXO EL NOMBRE D'EQUIPS QUE VAN A LA REPESCA INTERCONTINENTAL
            //SI S'ESCAU
            System.out.println(nEquips - sumEquipsConfederacio); 
        }   
    }
}
